opentreemap/treemap/audit.py

```diff
@@ -1391,12 +1391,19 @@ def short_descr(self):
 
         format_string = cls.action_format_string_for_audit(self)
 
-        model_display_name = cls.terminology(self.instance)['singular']
+        model_display_name = self.model_display_name()
 
         return format_string % {'field': self.field_display_name,
                                 'model': model_display_name.lower(),
                                 'value': self.current_display_value}
 
+    def model_display_name(self):
+        cls = get_auditable_class(self.model)
+        if hasattr(cls, 'display_name'):
```

Comment:
consider using `issubclass(cls, Convertible)`. I'm slightly concerned that because we have a `display_name` method on `Species` with a different signature, it could produce a tricky bug in the future.